tests/system/action/user/test_create.py

```diff
@@ -40,36 +42,27 @@ def test_create_some_more_fields(self) -> None:
                 "username": "test_Xcdfgee",
                 "default_vote_weight": "1.500000",
                 "organisation_management_level": "can_manage_users",
-                "guest_meeting_ids": [110, 111],
-                "committee_as_member_ids": [78],
-                "committee_as_manager_ids": [79],
+                "committee_ids": [78, 79],
```

Comment:
Is the CML already implemented?

Reply:
(there is an extra issue, but does the field exists already?)

Reply:
The general structure was implemented by @r-peschke, but currently the permission check would throw a `NotImplemented`. The field exists because I updated the models in this PR, but no logic for it is implemented yet.